packages/contracts/package.json

```diff
@@ -30,7 +30,7 @@
     "@unlock-protocol/networks": "workspace:^",
     "@unlock-protocol/tsconfig": "workspace:./packages/tsconfig",
     "@unlock-protocol/types": "workspace:^",
-    "chai": "4.3.7",
```

Comment:
Ha, should we remove it rather than update it?

Reply:
Actually here are like 25 other tests in the folder that uses chai currently, so it is a bit more work to remove it... We can either rename that PR and merge with all tests passing, or continue to work here to remove chai entirely. 

I personally dont think removing chai is a high priority as we removed the thing that was more pronve tp break (the `to.be.bignumber...` pattern).

I will advise we keep chai for the other and merge this PR as it is. Chai in itself is actually very stable